app/assets/stylesheets/config/_colors.scss

```diff
@@ -40,7 +40,7 @@
   --base-inverted: #fff;
 
   // Accent colors
-  --accent-brand: #3b49df;
+  --accent-brand: #3b49dd;
   --accent-brand-darker: #1827ce;
   --accent-brand-lighter: #8d95f2;
   --accent-brand-a10: #{rgba(#3b49df, 0.1)};
```

Comment:
If we move from `#3b49df ` to `#3b49dd`, should we change `--accent-brand-a10` as well in that case? (Of course, that is assuming that opacity passes accessibility tests!)

Reply:
I'm actually questioning whether that should change now that the color isn't in use in the footer or global-notice, which are the areas that I was trying to pass with WCAG. When the color is on top of white, it has a passing ratio. So I'm inclined to change it back and revisit if needed.